components/badge/badge.css

```diff
@@ -0,0 +1,114 @@
+/* parent selector */
+
+.badge {
+  position: relative;
+}
+
+/* Icon badges */
+
+.icon {
+  font-size: var(--size-12);
+  cursor: pointer;
+}
+
+.unit {
+  position: absolute;
+  border-radius: var(--radius-rounded);
+  min-width: var(--size-12);
+  height: var(--size-12);
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
I think this set of properties to Center the  item can be separated.

Reply:
I have added this to utilities.css, with a flexbox section. Thanks for recommendation